proposals/2020-fall/Agile-SIG.md

```diff
@@ -0,0 +1,43 @@
+# Application form for starting/continuing a SIG
+
+## What is/will be the name of the SIG
+Agile SIG
+
+## Which two persons will act as SIG Lead?
+- Berend Weel
+- Niels Drost
+
+## What is the SIG’s mission?
+
+The mission of the Agile SIG is to introduce, teach and train in Agile methodologies.
+We want Agile to become common knowledge in the center and aim for engineers/MT/DT/Operations to use it when appropriate.
+This will contribute to the team work within the center and to the restructuring and scaling phase we will enter in the near future.
+In addition the SIG acs as a platform for the exhange of experiences between the different teams at the center.
+
+## What is the GitHub repository of the SIG use?
+
+https://github.com/nlesc-sigs/Agile-SIG
+
+## What is the Office group of the SIG use?
+
+Agile-SIG@esciencecenter.nl (Agile SIG)
+
+## SIG outcomes in the period April 2020 - October 2020 (for existing SIGs)
+
+The Agile SIG has become the discussion platform for how to succesfully apply Agile methods to working at the eScience Center.
+Due to the Corona crisis, involvement of external people is postponed.
+
+We are also working on setting up an Introduction to Agile course. This is progessing nicely.
+
```

Comment:
Should we add the link? https://github.com/escience-academy/lesson-agile-escience

Reply:
yes, done